asyncscope.md

```diff
@@ -136,7 +136,9 @@ The requirements for the async scope are:
  - An `async_scope` must allow an arbitrary sender to be nested within the scope without eagerly starting the sender (`nest()`).
  - An `async_scope` must constrain `spawn()` and `spawn_future()` to accept only senders that are never-blocking.
  - An `async_scope` must constrain `spawn()` to accept only senders that complete with void.
- - An `async_scope` must provide an `empty_sender` that completes when all spawned senders are complete.
+ - An `async_scope` must provide an `@_on-empty-sender_@` that completes when all spawned senders are complete.
```

Comment:
One of the pieces of feedback was to add a bullet for the requirement that `spawn()` and `spawn_future()` must start the given sender before they exit.